client/js/components/day/day.js

```diff
@@ -18,8 +18,9 @@ var Day = React.createClass({
 
   componentWillMount: function(){
     DayStore.addChangeListener(this._handleUpdate);
-     // trigger API call to get initial data
-    ViewActionCreator.getDayData();
+     // trigger API call to get initial data. Date.now will
+     // change once we get routing sorted
+    ViewActionCreator.getDayData(Date.now());
```

Comment:
great point! i think ryan was saying we should use http://momentjs.com/ . we can definitely integrate that asap